### ISSUE TITLE
Tweet "date" information

### ISSUE THREAD

**Issue by gapmiss** [kbravh/tweet-to-markdown#10]
I noticed that tweet-to-markdown does not have the "date" information, which the Obsidian plugin has.

I edited the `main.js` directly. Here is the `diff` for anyone that might find this useful or if someone wants to upgrade the source code to add this feature.

```diff
17808c17808
<         'tweet.fields': 'attachments,public_metrics,entities,conversation_id,referenced_tweets',
---
>         'tweet.fields': 'attachments,public_metrics,entities,conversation_id,referenced_tweets,created_at',
18042a18043,18045
>   // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString
>   // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat
>   const cdate_options = { weekday: 'short', year: 'numeric', month: 'short', day: 'numeric', hour12: true,hour: 'numeric', minute: 'numeric', timeZone: 'America/Los_Angeles', timeZoneName: 'short' };
18043a18047,18048
>   let created = new Date(tweet.data.created_at);
>   let fetched = new Date();
18045a18051,18052
>           `date: ${created.toLocaleDateString('en-US', cdate_options)}`, // Tue, Aug 16, 2022, 2:41 PM PDT
>           `fetched: ${fetched.toLocaleDateString('en-US', cdate_options)}`,
18089a18097
>   const date_options = { year: 'numeric', month: 'short', day: 'numeric' };
18095c18103
<         `${user.name} ([@${user.username}](https://twitter.com/${user.username}))`, // name and handle
---
>         `${user.name} ([@${user.username}](https://twitter.com/${user.username})) - ${created.toLocaleDateString('en-US', date_options)}`, // name, handle and date (Aug 10, 2022)
```

**Comment by gapmiss** [comment]
## before:

![Screenshot 2022-08-16 at 22-51-41 ttm obsmd io](https://user-images.githubusercontent.com/98914514/185000616-e35201e4-ac81-4ff2-aef0-7fde9c3939fd.png)

## after:

![Screenshot 2022-08-16 at 22-53-49 ttm obsmd io](https://user-images.githubusercontent.com/98914514/185000653-440d8a64-1e04-42dd-85d1-a3248a373a73.png)

**Comment by kbravh** [comment]
I've just published version 2.4.0, which now includes the date of the tweet in the generated markdown. There's still some work to do to allow users to define their own formats, which I will follow here: https://github.com/kbravh/tweet-to-markdown/issues/17
